### PR TITLE
Let libetpan build on VS2013, VS2015, and VS2017.

### LINKED ISSUE #282: Not able to build in windows VS-2013:  'mailimf_fws_atom_for_word_parse' : must have external linkage in order to be exported/imported
hi,

I am getting the following error while building libetpan for windows:

c:\work\libetpan\src\low-level\imf\mailimf.c(1179): error C2201: 'mailimf_fws_atom_for_word_parse' : must have external linkage in order to be exported/imported.

### PR DESCRIPTION
**Pull request by eesheesh**
Also clear up a few warnings. I tested that it builds on Linux too, but I didn't check too thoroughly beyond that.

Fixes #282. Thanks @stringarray for identifying the solution.

### CODE REVIEW

File: src/low-level/imf/mailimf.c [hunk 1]
```diff
@@ -1174,8 +1174,8 @@ int mailimf_atom_parse(const char * message, size_t length,
 }
 
 LIBETPAN_EXPORT
```
Comment (dinhviethoa) [inline src/low-level/imf/mailimf.c]: Can we just remove LIBETPAN_EXPORT and leave "static"?

File: src/low-level/mime/mailmime_decode.c [hunk 2]
```diff
@@ -821,7 +821,9 @@ int mailmime_encoded_word_parse(const char * message, size_t length,
   mailmime_decoded_part_free(decoded);
 free_body:
   free(body);
+#if 0
```
Comment (dinhviethoa) [inline src/low-level/mime/mailmime_decode.c]: Could you remove this disabled code altogether?

**Comment by eesheesh** [comment]
@stringarray good catch, done.

@dinhviethoa made both of your changes.

I verified that this still builds on VS 2013, 2015, and 2017.

**Comment by eesheesh** [comment]
Thanks @dinhviethoa! Let me know if anything else is needed to merge this.

**Pull request merged**